core/trino-main/src/main/java/io/trino/operator/SimpleTableExecuteOperator.java

```diff
@@ -0,0 +1,148 @@
+/*
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package io.trino.operator;
+
+import com.google.common.collect.ImmutableList;
+import io.trino.Session;
+import io.trino.metadata.Metadata;
+import io.trino.metadata.TableExecuteHandle;
+import io.trino.spi.Page;
+import io.trino.spi.PageBuilder;
+import io.trino.spi.type.Type;
+import io.trino.sql.planner.plan.PlanNodeId;
+
+import java.util.List;
+
+import static com.google.common.base.Preconditions.checkState;
+import static io.trino.spi.type.BigintType.BIGINT;
+import static java.util.Objects.requireNonNull;
+
+public class SimpleTableExecuteOperator
+        implements Operator
+{
+    private static final List<Type> TYPES = ImmutableList.of(BIGINT);
+    private static final Page PAGE = new PageBuilder(1, TYPES).build();
```

Comment:
Remove `TYPES` and change to `new Page(0)` since this page is empty.

Reply:
Actually, is this supposed to return a row count? If so, then we need to build a page with a single row.

Reply:
we don't output row count for table EXECUTE  https://github.com/trinodb/trino/blob/9ab09e99d6c56fe955697a51a95400c367525133/core/trino-main/src/main/java/io/trino/sql/planner/LocalExecutionPlanner.java#L3924
reason is -- the execute internally can process whatever data (not necessarily table rows), and the row count of things processed is not necessarily meaningful. 

cc @losipiuk